.travis.yml

```diff
@@ -10,6 +10,11 @@ language: python
 python:
   - "2.7"
 
+addons:
+  apt:
+    packages:
+    - glpk-utils
```

Comment:
I mostly added this as documentation, because COFFEE now relies on this, even though its test suite does not actually uses it. I have no idea about Travis.